docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1475,7 +1475,7 @@ Other than using Alluxio FUSE daemon, you could also use CSI to mount the Alluxi
 In order to use CSI, you need a Kubernetes cluster with version at least 1.17, 
```

Comment:
For the CSI section, can you tell what's the differences between CSI and using helm/kubectl to deploy the Fuse daemon?